pkg/virt-launcher/virtwrap/live-migration-target.go

```diff
@@ -35,17 +37,97 @@ import (
 	"kubevirt.io/kubevirt/pkg/util/net/ip"
 	migrationproxy "kubevirt.io/kubevirt/pkg/virt-handler/migration-proxy"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/cli"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/converter"
 )
 
-func (l *LibvirtDomainManager) finalizeMigrationTarget(vmi *v1.VirtualMachineInstance) error {
+func (l *LibvirtDomainManager) finalizeMigrationTarget(vmi *v1.VirtualMachineInstance, options *cmdv1.VirtualMachineOptions) error {
+	interfacesToReconnect := interfacesToReconnect(options)
+	if len(interfacesToReconnect) != 0 {
+		if err := l.reconnectGuestNics(vmi, interfacesToReconnect); err != nil {
+			return err
+		}
+	}
+
 	if err := l.setGuestTime(vmi); err != nil {
 		return err
 	}
 
 	return nil
 }
 
+func interfacesToReconnect(options *cmdv1.VirtualMachineOptions) map[string]struct{} {
+	interfaceMigrationOptions := options.InterfaceMigration
+	ifacesToRefresh := map[string]struct{}{}
+	for ifaceName, migrationOption := range interfaceMigrationOptions {
+		if migrationOption != nil && migrationOption.Method == string(v1.LinkRefresh) {
+			ifacesToRefresh[ifaceName] = struct{}{}
+		}
+	}
+	return ifacesToRefresh
+}
+
+// Set interfaces link down and up to renew DHCP leases
```

Comment:
Function/Method comments need to start with the name of the func/method.

Reply:
done.